cmd/cloud-controller-manager/nodeipamcontroller.go

```diff
@@ -78,60 +78,56 @@ func startNodeIpamController(ccmConfig *cloudcontrollerconfig.CompletedConfig, n
 		return nil, false, fmt.Errorf("the AllocateNodeCIDRs is not enabled")
 	}
 
-	// the CloudAllocator ipam does not need service or cluster cidrs since it obtains the data from the cloud directly
```

Comment:
we need to add an unit test to `startNodeIpamController`  so we document all the inputs allowed, just to document what is allowed and what not

Reply:
Done